src/backend/access/aocs/aocssegfiles.c

```diff
@@ -1708,6 +1708,8 @@ aocol_compression_ratio_internal(Relation parentrel)
 
 	Oid			segrelid = InvalidOid;
 
+	Assert(Gp_role == GP_ROLE_DISPATCH);
```

Comment:
I think this is useless Assert, because this function is called from one place, where there is the same Assert.

Reply:
Ok, I placed it just in case once somebody may call it internally directly, but I guess it has a low probability. Then we'll need to remove the same assert from `aorow_compression_ratio_internal`

Reply:
done